modeling-cmds/src/def_enum.rs

```diff
@@ -809,6 +809,19 @@ define_modeling_cmd_enum! {
             pub curve_id: Uuid,
         }
 
+        /// Project an entity on to a plane.
```

Comment:
Could use a bit more detail or an example of its use.

Reply:
An example or in the comments of the endpoint? Projecting a point onto a plane is where you calculate the normal between a plane and a point that does not lie on the plane. You then move that point along the normal until it is now lying on the plane exactly.

Useful for constraints, shadows, intersection testing, etc.p